ubuntu-18.04/Dockerfile

```diff
@@ -34,6 +35,16 @@ RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so
 COPY docker-entrypoint.sh /
 RUN chmod 700 /docker-entrypoint.sh
 
+# Needed to support install of PostgreSQL 12 and MySQL 5.7
+RUN which curl || apt -y install curl
+RUN export DEBIAN_FRONTEND=noninteractive
+RUN echo exit 0 > /usr/sbin/policy-rc.d
+RUN apt-get install -y openssh-server netbase perl libnuma1 psmisc  locales locales-all libxslt1.1
```

Comment:
Should this be the responsibility of the postgres / mysql blueprint to do, or should we hard-bake it into the base image in the assumption that end-users would have done this in a real environment?

It seems wrong to install too much into the base docker image.

Thoughts?